### DISCUSSION TITLE
Missing .nuclei-ignore file

### DISCUSSION THREAD

**Discussion by dotfy** [projectdiscovery/nuclei#1835] [Q&A] [answered]
`[ERR] Could not read nuclei-ignore file: open /root/.config/nuclei/.nuclei-ignore: no such file or directory`

My templates are up to date. but i am facing this problem. How can I fix?

**Comment by forgedhallpass** [comment, marked as answer]
Hello @dotfy,

I wonder if you've managed to figure out what the issue was. Normally nuclei updates that file automatically.

At this point in time nuclei looks for the `.nuclei-ignore` file under the configuration folder, which by default is under `$HOME/.config/nuclei/`, or under the user provided custom directory, in case it has been overwritten using the `-config` flag. If it cannot find it under any of those folders, it will check within the current working directory.

If you execute nuclei with both normal user and administrative privileges, with the default configuration, it will create the configuration directory for both users. (Please note that nuclei doesn't require administrative privileges)

You can try to remove the configuration directory altogether and re-try the execution.

Alternatively, you can also try manually download it from [here](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore) and placing it under one of the above mentioned locations, but make sure to provide the right permissions.

**Reply by amr-ShaglouF** [reply]
@forgedhallpass 
I asking if you have video explain this steps

**Comment by prakadesh** [comment]
how to solve this issue in docker installation method

**Reply by galoget** [reply]
You can solve the issue by copying `.nuclei-ignore` file from your host into the Docker Container.

In a terminal, follow these steps:

1. Download `.nuclei-ignore` file in your host home directory:

```bash
wget https://raw.githubusercontent.com/projectdiscovery/nuclei-templates/master/.nuclei-ignore -O $HOME/.nuclei-ignore
```

2. Get the container ID of your current running container:

```bash
sudo docker ps
```

3. Use that ID to copy `.nuclei-ignore` from your host into the container:

```bash
sudo docker cp $HOME/.nuclei-ignore container_id:/root/.config/nuclei/.nuclei-ignore
```

You can also try other methods mentioned [here](https://www.baeldung.com/ops/docker-copying-files).

Hope it helps!

**Comment by prakadesh** [comment]
![image](https://user-images.githubusercontent.com/84655612/209296685-1b9be818-f760-4efc-8183-fcb5b95faa0f.png)

**Reply by Steiner-254** [reply]
Did you manage to fix that issue?
